libs/components/src/shared/patterns/form-elements.ts

```diff
@@ -59,3 +62,16 @@ export function formElements<T extends { new (...args: any[]): Record<string, an
 
 	return Decorated;
 }
+
+export const errorMessageTemplateFactory: (context: ElementDefinitionContext) =>
+() => ViewTemplate<FormElement> =
+	(context) => {
+		const iconTag = context.tagFor(Icon);
+		return () => {
+			return html<FormElement>`
+				${when(x => x.errorValidationMessage, html<TextField>`
+					<${iconTag} class="error-message-icon" type="info-negative"></${iconTag}>
+					<span class="error-message">${x => x.errorValidationMessage}</span>`)}
```

Comment:
```suggestion
<div class="base">
					<${iconTag} class="icon" type="info-negative"></${iconTag}>
					<span class="message">${x => x.errorValidationMessage}</span>`)}
</div>
```
and I would add CSS:
```
.base {display: flex; row-gap: 8px; align-items: flex-start;}
.icon {flex-shrink: 0;}
```